internal/pkg/manifest/validate.go

```diff
@@ -1498,6 +1498,15 @@ func (s SidecarConfig) validate() error {
 	}
 	return s.ImageOverride.validate()
 }
+func (s SidecarConfig) validateSidecarImageFields() error {
+	if s.Image.IsZero() {
+		return fmt.Errorf(`must specify one of "image", "image.build, or "image.location"`)
+	}
+	if err := s.Image.validate(); err != nil {
+		return fmt.Errorf(`validate "image.build": %w`, err)
```

Comment:
I thought `s.Image.validate()` would validate both `image.location` and `image.build` right?
```suggestion
		return fmt.Errorf(`validate "image": %w`, err)
```

Reply:
That's true 👍